components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,221 @@
++++
+title = "ServiceNow Integration"
+
+draft = false 
+
+gh_repo = "automate"
+
+[menu]
+  [menu.automate]
+    title = "ServiceNow Integration"
+    parent = "automate/reference"
+    identifier = "automate/reference/servicenow_integration.md ServiceNow Integration"
+    weight = 50
++++
+
+## Chef Automate Integration with CMDB App in ServiceNow
```

Comment:
I wrote up an introduction to explain this section:

[ServiceNow](https://www.servicenow.com/) provides cloud-based services that automate enterprise IT operations. ServiceNow specializes in IT service management (ITSM) applications and provides forms-based workflow application development. ServiceNow supports third-party application and data integrations. The most common integrations are with configuration management database (CMDB), incident management, problem management, change management, user administration, and single sign-on authentication.

The Chef Automate plugin for ServiceNow enables the integration of Chef Automate in the Service now ecosystem. The Chef Automate integration is a ServiceNow-certified scoped application available from the ServiceNow store. It integrates existing Chef Automate infrastructure and compliance functionality with ServiceNow enterprise services. Once installed and configured, this integration allows Chef Automate to create and update a ServiceNow configuration management database (CMDB) with data from nodes managed by Chef Automate.

Reply:
Thanks Kim. I have used your write-up as is with very few changes!